Kernel/Bus/PCI/Device.cpp

```diff
@@ -5,15 +5,20 @@
  */
 
```

Comment:
Commit message:
* "If the PCI device do not" -> "If the PCI device does not"
* "can be used by the actually allocate an irq" -> what?
* irq -> IRQ
* pin based -> pin-based